tests/plotting/test_plotting.py

```diff
@@ -2491,8 +2492,9 @@ def test_add_text():
 
 
 @pytest.mark.skipif(
-    not vtk.vtkMathTextFreeTypeTextRenderer().MathTextIsSupported(),
-    reason='Math text is not supported.',
+    tuple(map(int, matplotlib.__version__.split('.')[:2])) >= (3, 6)
+    and pyvista.vtk_version_info <= (9, 2, 2),
```

Comment:
Well... I think we still want the `not vtk.vtkMathTextFreeTypeTextRenderer().MathTextIsSupported()` check in there too... Though this is almost always true if Matplotlib is installed. Unless `VTK_MODULE_ENABLE_VTK_RenderingMatplotlib` is not enabled.... which my custom wheels have disabled 🤦🏻‍♂️

Reply:
> which my custom wheels may have disabled
Ugh yep: https://github.com/banesullivan/vtk-cmake/blob/b1eef4ca2c6bbb2e018406086df83f7e42e4af19/cmake/_modules.cmake#L26